canvas_modules/common-canvas/package.json

```diff
@@ -20,9 +20,10 @@
   },
   "dependencies": {
     "@elyra/pipeline-schemas": "3.0.13",
-    "@carbon/icons-react": "10.8.2",
-    "carbon-components": "10.9.3",
-    "carbon-components-react": "7.9.3",
+    "@carbon/icons-react": "10.13.0",
+    "carbon-components": "10.14.0",
+    "carbon-components-react": "7.14.0",
+    "carbon-icons": "7.0.7",
```

Comment:
The tests were failing and I traced it to <Icon> from Carbon Components needing `carbon-icons` to run properly. There is no package.json in `carbon-components` requiring it but it will fail without it.

Reply:
If that line is removed, the tests will fail with this new version of `carbon-components`

Reply:
Also, the new carbon-components version added color styling to ghost buttons, which originally was overridden by the `fill` we set, but now needs to be manually overridden specifically for bx-button components